cmd/scan/domains/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
The script expects a folder, and this is the only way to create an empty folder in git. :-(

I'll probably change the script to be more flexible at some point, but I think this is okay for now.